server/test_helpers/updateTestDataSqlite.ts

```diff
@@ -0,0 +1,71 @@
+// This file is deprecated.
```

Comment:
What is the goal of this file?

Reply:
@sandoche Originally I thought I'd use sqlite, but if I get Postgres working for the tests, that will be better since the prod indexer uses pg too, so then I'll delete the sqlite stuff.